skycoin/skywire
# VPN Control Buttons in Systray

Did you run `make format && make check`? Yes

Fixes #860 

 Changes:	
- Add VPN control button on systray:
  - VPN connection status
  - Select server from list
  - Connect/Disconnect button
- Remove transport cache and get it before create route

How to test this PR:
- Needs to install systray dependencies, follow instruction on [the docs](https://github.com/mrpalide/skywire/blob/feature/improve-systray/docs/systray-builds.md)
- `make build-systray` [or `make build-systray-windows` for Windows]
- Run visor
  - **note:** In Windows machine visor should run as administrator to make routes for VPN client. But in *nix machine, please set `cap_net_admin` by `sudo setcap 'cap_net_admin+p' ./apps/vpn-client` command and run visor normal (without root permission).
- Run VPN server on test env, and choose it from list and use/test other buttons
